torch/_prims_common/__init__.py

```diff
@@ -634,6 +634,15 @@ def extract_shape(*args, allow_cpu_scalar_tensors: bool) -> Optional[ShapeType]:
     return shape if shape is not None else scalar_shape
 
 
+def extract_dims_from_varargs(dims: Union[DimsSequenceType, Tuple[DimsSequenceType, ...]]) -> DimsSequenceType:
```

Comment:
Nice -- maybe there's a short comment this PR can add to describe this function, too